### DISCUSSION TITLE
Nothing happend since a long time. Is there an equivalent project ?

### DISCUSSION THREAD

**Discussion by pedefe** [android-js/androidjs#229] [General]
Hello,

This project seems very good, very simple to use.
But it seems that nothing happens since a long time.

Does somebody knows about that, or about an equivalent project ?

Best regards.

**Comment by Chhekur** [comment]
Hey @pedefe 
we are actually working as a full-time software developer so didn't get a lot of time to contribute to the project.
so we are looking for a community to contribute to the project still we can guide us about how to add new things to this and make this better.

**Reply by pedefe** [reply]
thank you very much.
